### PR TITLE
zh-trans: add /docs/tasks/configure-pod-container/configure-projected…

### PR DESCRIPTION
**Pull request by AdamDang**
…-volume-storage.md

zh-trans: add /docs/tasks/configure-pod-container/configure-projected-volume-storage.md

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+       echo -n "1f2d1e2e67df" > ./password.txt
+
+    # Package these files into secrets:
+       kubectl create secret generic user --from-file=./username.txt
+       kubectl create secret generic pass --from-file=./password.txt
+```
+-->
+
+1. 创建 Secrets:
+```shell
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md]: should we indent line 75 and similar lines below to make sure the ordered list on line 62 is not broken?

**Comment by tengqm** [comment]
Translation is very good. Just one doubt about the ordered list. Please double check in the preview.

File: content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md (hunk 1 above)
Comment (AdamDang) [reply content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md]: I tried just now and found not solved the ordered list problem.

Comment (tengqm) [reply content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md]: https://5c26e3c33bfbcc000b754519--k8s-v1-12.netlify.com/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage/

File: content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+-->
+
+{{% capture overview %}}
+<!--
+This page shows how to use a [`projected`](/docs/concepts/storage/volumes/#projected) volume to mount
+several existing volume sources into the same directory. Currently, `secret`, `configMap`, `downwardAPI`,
+and `serviceAccountToken` volumes can be projected.
+-->
+
+本文介绍怎样通过[`投射（projected）`](/docs/concepts/storage/volumes/#projected) 卷将现有的多个卷资源挂载到相同的目录。
```
Comment (chenrui333) [inline content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md]:
```suggestion
本文介绍怎样通过[`投射`](/docs/concepts/storage/volumes/#projected) 卷将现有的多个卷资源挂载到相同路径下。
```

File: content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
+---
+reviewers:
+- jpeeler
+- pmorie
+title: 配置 Pod 使用投射卷存储
```
Comment (chenrui333) [inline content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md]:
```suggestion
title: 配置 Pod 使用投射卷作存储
```

File: content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+## Configure a projected volume for a pod
+
+In this exercise, you create username and password Secrets from local files. You then create a Pod that runs one Container, using a [`projected`](/docs/concepts/storage/volumes/#projected) Volume to mount the Secrets into the same shared directory.
+
+Here is the configuration file for the Pod:
+-->
+
+## 为 Pod 配置投射卷
+
+本练习中，您将从本地文件来创建包含有用户名和密码的 Secret。然后创建运行一个容器的 Pod，该 Pod 使用[`投射（projected）`](/docs/concepts/storage/volumes/#projected) 卷将 Secret 挂载到相同的共享目录。
```
Comment (chenrui333) [inline content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md]:
```suggestion
本练习中，您将从本地文件来创建包含有用户名和密码的 Secret。然后创建运行一个容器的 Pod，该 Pod 使用[`投射`](/docs/concepts/storage/volumes/#projected) 卷将 Secret 挂载到相同的路径下。
```

File: content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+-->
+1. 创建 Pod：
+```shell
+       kubectl create -f https://k8s.io/examples/pods/storage/projected.yaml
+```
+<!--
+1. Verify that the Pod's Container is running, and then watch for changes to
+the Pod:
+-->
+1. 确认 Pod 中的容器运行正常，然后监视 Pod 的变化：
```
Comment (chenrui333) [inline content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md]: 你这里可能需要注意 markdown 序列号的问题。
带序号的最好是不要隔开，不然不能构成序列。

File: content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+
+<!--
+* Learn more about [`projected`](/docs/concepts/storage/volumes/#projected) volumes.
+* Read the [all-in-one volume](https://github.com/kubernetes/community/blob/{{< param "githubbranch" >}}/contributors/design-proposals/node/all-in-one-volume.md) design document.
+-->
+
+* 进一步了解[`投射（projected）`](/docs/concepts/storage/volumes/#projected) 卷。
```
Comment (chenrui333) [inline content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md]:
```suggestion
* 进一步了解[`投射`](/docs/concepts/storage/volumes/#projected) 卷。
```

File: content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md (hunk 3 above)
Comment (AdamDang) [reply content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md]: Done！

File: content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md (hunk 2 above)
Comment (AdamDang) [reply content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md]: Done！

File: content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md (hunk 4 above)
Comment (AdamDang) [reply content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md]: Done！

File: content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md (hunk 6 above)
Comment (AdamDang) [reply content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md]: Done！

File: content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md (hunk 5 above)
Comment (AdamDang) [reply content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md]: agree！ 译文直接放在后面了。

**Comment by AdamDang** [comment]
Thanks for reviewing！

**Pull request merged**